test/track.py

```diff
@@ -11,9 +11,11 @@ class Track:
     tags = attr.ib(factory=list)
 
     @classmethod
-    def create(cls, recording):
+    def create(cls, recording, start_s=10):
         "Make a TestTrack with some plausible data."
-        return cls(None, recording, data={"foo": [[1, 2], [3, 4]], "start_s": 10, "end_s": 20})
+        if start_s is None:
+            start_s = 10
+        return cls(None, recording, data={"foo": [[1, 2], [3, 4]], "start_s": start_s, "end_s": start_s + 10})
```

Comment:
Is this actually plausible data for a track?  Don't we have something more formally defined?  Is the `foo` property just freeform data, or something else?

Reply:
I'm guessing this is there, just to test that any data can be entered in there. there are probably other tests that rely on this.
Should really be removed and use more expected data like start_s and end_s